CMakeLists.txt

```diff
@@ -191,9 +199,19 @@ if(${BUILD_ATTACH_IMPL_EXAMPLE})
 endif()
 
 # benchmark that requires bpftime libraries
+if(${BPFTIME_BUILD_WITH_LIBBPF})
+# Currently benchmark is using libbpf
 add_subdirectory(benchmark)
+endif()
 
 
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
 
-install(TARGETS bpftime-agent bpftime_text_segment_transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ~/.bpftime)
+set(DEST_DIR "$ENV{HOME}/.bpftime")
+if(${BPFTIME_BUILD_WITH_LIBBPF})
+    install(TARGETS bpftime-agent bpftime_text_segment_transformer bpftime-syscall-server CONFIGURATIONS Release Debug RelWithDebInfo DESTINATION ${DEST_DIR})
+elseif(APPLE)
```

Comment:
Is there any difference between the `elseif` and the `else` branch? I can't spot any

Reply:
i had it written for a different rule for APPLE, but by the end it was same for else and elseif, i will combine them. Thanks for pointing out.